### ISSUE TITLE
SVG rendering in Jupyterlab doens't work for Firefox

### ISSUE THREAD

**Issue by biermeester** [xflr6/graphviz#52]
I noticed that the SVG graphs wouldn't render properly in Jupyterlab under Firefox.

Some investigation and testing revealed that the `svg+xml` returned by the built-in `_repr_svg_` methods does not URL encode the `#` characters. This is apparently something that is required by the specifications, although Firefox seems to be the only one refusing to render the data. (See https://stackoverflow.com/questions/30733607/svg-data-image-not-working-on-firefox?answertab=active#tab-top for reference)

Adding a simple `.replace('#', '%23')` to the output of `_repr_svg_` seems to resolve the problem, although there might be a more elegant solution that I'm not aware of.

**Comment by xflr6** [comment]
Thanks. Trying the `replace` fix in d7fba7355261e4c335088d04420fbd6938397667. Maybe better use [urllib.parse.quote](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.quote) to do this.

**Comment by xflr6** [comment]
Oh. I think this is a Jupyterlab issue. Inline SVG like the following does actually work in Jupyter notebook but not in Jupyterlab (please report there):
```python
%matplotlib inline
%config InlineBackend.figure_format = 'svg'

import numpy as np
import matplotlib.pyplot as plt

t = np.arange(0.0, 2.0, 0.01)
s = 1 + np.sin(2 * np.pi * t)

fig, ax = plt.subplots()
ax.plot(t, s)

ax.set(xlabel='time (s)', ylabel='voltage (mV)',
       title='About as simple as it gets, folks')
ax.grid()

plt.show()
```
![jupyterlab](https://user-images.githubusercontent.com/6342379/41893358-c8e64adc-791b-11e8-99fd-c0ea083e63ab.png)
